<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSRC</class>
 <widget class="QWidget" name="WSRC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WSRC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 11, 11);</string>
  </property>
  <widget class="QScrollArea" name="Remote_Area">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>871</width>
     <height>620</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>869</width>
      <height>618</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="remote_terminal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>869</width>
       <height>619</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 127);</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="output_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>远程终端</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="send_thing">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="My_Area">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>140</y>
     <width>561</width>
     <height>621</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>559</width>
      <height>619</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="my_terminal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>558</width>
       <height>619</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 127);</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
